apps/internal/oauth/ops/accesstokens/accesstokens.go

```diff
@@ -92,6 +92,9 @@ type Credential struct {
 	// Key is the private key for signing if we are doing any auth other than secret.
 	Key crypto.PrivateKey
 
+	// SendX5C specifies if x5c claim(public key of the certificate) should be sent to STS
+	SendX5C bool
```

Comment:
This isn't really part of the credential. Would this make more sense to be modeled as part of AuthParams? thoughts?

Reply:
I think semantically, makes more sense in AuthParams.  

My caveat to that is how hard it is to add to authParams vs where it is. If that starts to get messy, I'd just leave it.

Reply:
Added it in AuthParams in this commit https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/192/commits/77406fefb57f82aad9027449990cd080a67a7adf. Does not get that messy. So I think we can keep it in AuthParams. Please take a look.